Airports/EGBB/RMA.txt

```diff
@@ -51,4 +51,11 @@ Birmingham Delegation       S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00
                             N052.14.30.933 W001.47.58.967 N052.14.27.637 W001.47.14.911
                             N052.14.27.637 W001.47.14.911 N052.14.25.280 W001.46.30.697
                             N052.14.25.280 W001.46.30.697 N052.14.23.864 W001.45.46.375
-                            N052.14.23.864 W001.45.46.375 N052.14.23.000 W001.44.42.000
\ No newline at end of file
+                            N052.14.23.864 W001.45.46.375 N052.14.23.000 W001.44.42.000
+Birmingham Buffer           S999.00.00.000 E999.00.00.000 S999.00.00.000 E999.00.00.000
```

Comment:
maybe title this the 'Birmingham/ East Midlands Buffer'

Reply:
Your suggestion wouldn't fit the current indentation so I changed it to BB/NX Buffer if that's any better. Could always just leave it as it was or increase the indentation, lmk.

Reply:
I think Birmingham Buffer works (sorry to be a pain 😂)

Reply:
Nw